### PR TITLE
schutzbot: simplify CI runs a bit

### PR DESCRIPTION
**Pull request by teg**
Don't build osbuild (except for when testing against 8.2), so that we test against the dependencies that we have already landed.

### CODE REVIEW

**Review by msehnout — COMMENTED** [review]
Looks good to me :+1: . I've restarted the CI hoping that we can get "greener" result.

**Comment by msehnout** [comment]
It seems that RHEL 8.3 base, integration, and OSTree pipelines are consistently unhappy with this PR:
https://osbuildci.cloud.paas.psi.redhat.com/blue/organizations/jenkins/osbuild%2Fosbuild-composer/detail/PR-966/2/pipeline/
https://osbuildci.cloud.paas.psi.redhat.com/blue/organizations/jenkins/osbuild%2Fosbuild-composer/detail/PR-966/3/pipeline/
https://osbuildci.cloud.paas.psi.redhat.com/blue/organizations/jenkins/osbuild%2Fosbuild-composer/detail/PR-966/4/pipeline

**Comment by ondrejbudai** [comment]
@teg Do we still want this? Please rebase/close accordingly. :slightly_smiling_face:

**Review by atodorov — APPROVED** [review]
LGTM too but needs rebase

**Comment by teg** [comment]
Sorry for taking so long to get back to this. Rebased. It might be that we can only merge this once the next `osbuild` has landed in fedora though.

**Review by ondrejbudai — APPROVED** [review]
Looks good!

**Comment by ondrejbudai** [comment]
@jkozol This might cause you some issues on #1013 as osbuild is not yet rebased on 8.4.
@gicmo when are you going to rebase osbuild on 8.4?

**Comment by teg** [comment]
> @jkozol This might cause you some issues on #1013 as osbuild is not yet rebased on 8.4.
> @gicmo when are you going to rebase osbuild on 8.4?

Hm, good point. On the one hand: good! We don't want to accidentally push packages into 8.4 that would break it. However, we DO actually want to run the koji tests of building of 8.4 on top of 8.3 as that's how it will be used. Schutzbot doesn't yet make this convenient to configure though.

**Comment by jkozol** [comment]
> @jkozol This might cause you some issues on #1013 as osbuild is not yet rebased on 8.4.

Good point. When I rebase on this I will expand the conditional so it builds from the submodule for 8.3 and 8.4.

**Comment by teg** [comment]
> > @jkozol This might cause you some issues on #1013 as osbuild is not yet rebased on 8.4.
> 
> Good point. When I rebase on this I will expand the conditional so it builds from the submodule for 8.3 and 8.4.

Oh, actually @gicmo mentioned that osbuild is already in 8.4, so this might be good already.

**Comment by larskarlitski** [comment]
GitHub closed this pull request because I renamed osbuild's main branch of development to `main`, deleting `master`. If it is still relevant, please reopen it against the `main` branch. Thank you!

**Pull request closed without merging**
